contracts/ICS20Transfer.sol

```diff
@@ -142,6 +147,49 @@ contract ICS20Transfer is
         );
     }
 
+    /// @inheritdoc IICS20Transfer
```

Comment:
I wasn't able to see a better way of doing this that didn't increase gas cost for the non-permit transfer case. Let me know if you can see a way that is less repetitive.